### DISCUSSION TITLE
Not receiving rewards since 1.8.0

### DISCUSSION THREAD

**Discussion by lukeisbackontrack** [Chia-Network/chia-blockchain#15405] [Support]
I run solo farm 1.53PiB I used to received rewards every 3 days or so, since installed 1.8.0 I haven’t received a reward. Now running 1.8.1. Last time I received a reward was 7th of May. There aren’t any errors in the log apart from drive related warning that specific directory doesn’t exist. I first didn’t make much of it and thought it’s due to bad luck, but it’s end of May and I didn’t receive a reward since 7th May. I farm solo, using GUI. Looking at chia explorer my full node is up to date. I don’t have any other choice, I will have to pull the plug, electricity bill too expensive.

**Comment by iTZUNAMI** [comment]
in april there were 19-20 EiB on total mainet, now about 23.527 EiB so 10%-20% time more to wait, but yes also my last block i found was on 7 april, usually i get 1 every 28 days and now i'm waiting too after the switch...
Also on 1.8.0 there was a securty hole and they didn't say nothing after that:

Critical Security patches

    This release includes patches that are critical security updates for all network users. It is highly recommended that all network nodes upgrade to the 1.8.0 version.
    We will be publishing a post mortem in the near future to discuss the specific issues we are patching with this release, until then, responsible disclosure dictates that we not address the issues directly.
    Please monitor our official communication streams for more information on this matter once we feel it is safe to disclose.
